packages/test/src/test-local-activities.ts

```diff
@@ -10,30 +10,40 @@ import { RUN_INTEGRATION_TESTS } from './helpers';
 import * as workflows from './workflows/local-activity-testers';
 
 interface Context {
+  workflowBundle: WorkflowBundleWithSourceMap;
   taskQueue: string;
   client: WorkflowClient;
+  getWorker: () => Promise<Worker>;
 }
 
 const test = anyTest as TestInterface<Context>;
 
+test.before(async (t) => {
+  t.context = {
+    workflowBundle: await bundleWorkflowCode({ workflowsPath: require.resolve('./workflows/local-activity-testers') }),
+  } as Context;
```

Comment:
@bergundy Like this? (Without the cast, the other props are optional, which adds a lot of `?.` to the tests.)

Reply:
`t.context.workflowBundle = ...`